MAUI/NumericEntry/Migration.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Migrating from Xamarin to .NET MAUI SfNumericEntry | Syncfusion 
+description: Learn about Migrating from Syncfusion Xamarin NumericTextBox and NumericUpDown to Syncfusion .NET MAUI NumericEntry control and more.
+platform: maui
+control: SfNumericEntry
+documentation: ug
+---  
+
+# Migrating from Xamarin SfNumericTextBox and SfNumericUpDown to .NET MAUI SfNumericEntry
+
+To migrate easier from the Xamarin SfNumericTextBox and SfNumericUpDown to `.NET MAUI SfNumericEntry`, most of the APIs are kept from the Xamarin SfNumericTextBox and SfNumericUpDown  in MAUI SfNumericEntry. However, to maintain the consistency of API naming in MAUI SfNumericEntry, rename some of the APIs. The APIs changed in MAUI SfNumericEntry from Xamarin SfNumericTextBox and SfNumericUpDown are detailed as follows.
```

Comment:
To migrate easier from the Xamarin SfNumericTextBox and SfNumericUpDown to `.NET MAUI SfNumericEntry`, most of the APIs are kept from the Xamarin SfNumericTextBox and SfNumericUpDown in MAUI SfNumericEntry. However, rename some of the APIs to maintain the consistency of API naming in MAUI SfNumericEntry. The APIs changed in MAUI SfNumericEntry from Xamarin SfNumericTextBox and SfNumericUpDown are detailed as follows.

Reply:
changes are done